src/components/main/NewItemButton.tsx

```diff
@@ -45,6 +45,7 @@ const NewItemButton = ({
             background: theme.palette.primary.main,
             '&:hover': { background: 'grey' },
           }}
+          data-umami-event="new-item-icon-button"
```

Comment:
Add page? or context? could be add in home or in children.

Reply:
Do we really need to know if it was in children or in home ?